includes/degree-functions.php

```diff
@@ -501,8 +505,21 @@ function get_degree_course_overview_modern_layout( $degree ) {
 						</div>
 					</div>
 				</div>
+				<div class="row">
+					<div class="col-12">
+						<a href="<?php echo $post_meta['degree_pdf']; ?>" rel="nofollow">
```

Comment:
I don't think all of our programs have a catalog URL available...while the degrees we're setting up with this template will all likely have one set, it might be safest to check and make sure `$post_meta['degree_pdf'];` isn't empty before displaying this link.